client/lib/redux-bridge/index.js

```diff
@@ -25,3 +31,21 @@ export function reduxDispatch( ...args ) {
 	}
 	reduxStore.dispatch( ...args );
 }
+
+function markedFluxAction( action ) {
+	return Object.assign( {}, action, { type: `FLUX_${ action.type }` } );
+}
+
+// this is a Map<ActionType:string, transform:action=>action
+const forwardActionMap = new Map();
+forwardActionMap.set( 'RECEIVE_MEDIA_ITEM', markedFluxAction );
+forwardActionMap.set( 'RECEIVE_MEDIA_ITEMS', markedFluxAction );
+
+function forwardAction( fluxAction ) {
+	const transform = forwardActionMap.get( fluxAction.type );
+	if ( transform ) {
+		reduxDispatch( transform( fluxAction ) );
```

Comment:
should we always change the action type to be `FLUX_${ action.type }`?

Reply:
It looks like a good default. It shouldn't be hard to add another `case` statement into a reducer:
```js
case RECEIVE_MEDIA_ITEM:
case FLUX_RECEIVE_MEDIA_ITEM:
  handleThemBoth();
```
And we can always write a custom transformer if neccessary.